src/components/RuleBuilder/utils/export.js

```diff
@@ -36,11 +36,11 @@ const exportRules = (rules, options) =>
         return parseInt(value, 10)
       }
 
-      const split = value.split('.')
-      return [parseInt(split?.shift(), 10), parseInt(split?.pop(), 10)]
+      const split = value.split(',')
+      return `${parseInt(split?.shift(), 10)},${parseInt(split?.pop(), 10)}`
```

Comment:
Why is this splitting and formatting necessary? We already have the value in the `left,right` format, right?

Reply:
94f07e8e31ff891ff2542e8f5f6633ce0cf3b32c